apps/console/src/extensions/configs/attribute.ts

```diff
@@ -131,6 +131,6 @@ export const attributeConfig: AttributeConfig = {
         isSCIMCustomDialectAvailable: () =>  { return Promise.resolve(""); },
         isUserStoresHidden: () =>  { return Promise.resolve([]); },
         mapClaimToCustomDialect: false
-    }, 
-    showCustomDialectInSCIM: false
+    },
+    showCustomDialectInSCIM: true
```

Comment:
Hope this change is intentional @thivi

Reply:
Yes. Setting this to `false` hid the custom attributes in the attribute mappings page. This has to be said to `true` to show them. @DimalChandrasiri, can you confirm if this is fine?

Reply:
I am reverting this to the previous state as we need to have a discussion on if this should be available here. If this should be set to `false`, then we should also disable the `Add Attribute Mapping` button in the Attributes pages.